test/tests/amqp/test_amqp_racknode.py

```diff
@@ -0,0 +1,211 @@
+'''
+Copyright 2017 Dell Inc. or its subsidiaries.  All Rights Reserved.

```

Comment:
it should be 'Copyright 2016, Dell EMC, Inc.'  'Dell EMC' is the 'its subsidiaries'